wiki/Tournaments/MWC/2014/es.md

```diff
@@ -1,143 +1,112 @@
 ---
-outdated_translation: true
+tags:
+  - MWC 2014
+  - MWC 4K 2014
+  - MWC 7K 2014
+  - MWC2014
+  - MWC4K 2014
+  - MWC7K 2014
 ---
 
 # osu!mania World Cup 2014
 
 ![MWC 2014 logo](img/logo.jpg)
```

Comment:
image tooltips and alt-texts arent translated in the entire file

for reference

`![alt-text](image-link "tooltip")`